src/components/SearchBar.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+export default class SearchBar extends Component {
+  render() {
+    return (
+      <div className="searchBox vertical">
+        <p className="searchLabel">FIND YOUR MOVIE</p>
```

Comment:
Just FYI. The best practice is keep various labels in constant, so you will be able to use these constants across the app. And when you need to change the label, you can change it in one place not be afraid you forgot to change somewhere in your code. In that case you can declare constants below the imports:
```
const searchLabel = 'FIND YOUR MOVIE';
...
...
<p className="searchLabel">{searchLabel}</p>

```

Reply:
Done.